docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+JDBC Poller Input Operator
+=============
+
+## Operator Objective
+This operator scans jdbc table in parallel fashion. The jdbc table is table in jdbc complaint database. To provide parallelism during read each instance of Apex input operator also known as operator partition reads one portion of jdbc table. Based on requested operator instances number of non-polling static partitions are created to scan table portitions. The non-polling partition will scan the table portion only once. Additional single operator partition is created to scan newly added data incrementally. 
+
+#### Assumption
+Assumption is that there is an ordered column using which range queries can be formed. That means database has a column which has unique constraint as well as the newly inserted record should have column value more than max value in that column.
```

Comment:
Generally speaking, we should anticipate and answer all the questions that are likely to occur to the reader. Here
are some to come to mind in this context:
1. Why is this assumption needed ?
2. Does it have to be a single column or can it be a combination of columns ?
3. Why max value (i.e. why not min value) ?

Also, if the intent is to say that every new record is constrained in some way, "the newly inserted" should
read "every newly inserted".